src/lib.zig

```diff
@@ -1357,6 +1357,14 @@ pub const Lua = struct {
         return c.lua_objlen(lua.state, index);
     }
 
+    /// returns length of table in any lua version
+    pub fn versionAgnosticLen(lua: *Lua, index: i32) usize {
```

Comment:
I like the intention of this function (I meant to add one soon), but I don't love the name.

I took a look at the [LPeg](https://www.inf.puc-rio.br/~roberto/lpeg/) source code. Internally for compatibility between Lua versions Roberto (the creator of Lua) aliases rawlen to objlen. So I think we can trust that those two are compatible enough :)

So could you instead modify the `rawLen` function to call objlen for lua51 and luau? I'm also happy to do this myself after merging the PR

Reply:
Sure I can do this